src/startupscriptgenerator/src/php/scriptgenerator.go

```diff
@@ -29,8 +29,15 @@ func (gen *PhpStartupScriptGenerator) GenerateEntrypointScript() string {
 	scriptBuilder.WriteString("#!/bin/sh\n")
 	scriptBuilder.WriteString("# Enter the source directory to make sure the script runs where the user expects\n")
 	scriptBuilder.WriteString("cd " + gen.SourcePath + "\n")
-	scriptBuilder.WriteString("export APACHE_DOCUMENT_ROOT='" + gen.SourcePath + "'\n")
 	common.SetEnvironmentVariableInScript(&scriptBuilder, "APACHE_PORT", gen.BindPort, DefaultBindPort)
+	common.SetEnvironmentVariableInScript(&scriptBuilder, "NGINX_PORT", gen.BindPort, DefaultBindPort)
```

Comment:
Can you add end to end tests covering this scenario?

Reply:
end to end tests are coming .. added some runtime